scripts/pi-hole/js/settings-advanced.js

```diff
@@ -288,41 +299,38 @@ function createDynamicConfigTabs() {
     url: "/api/config?detailed=true",
   })
     .done(function (data) {
-      // Create the content for the advanced dynamic config topics
+      // Create the tabs for the advanced dynamic config topics
       Object.keys(data.topics).forEach(function (n) {
         var topic = data.topics[n];
-        $("#advanced-content").append(
-          '<div class="col-lg-12" id="advanced-content-' +
-            topic.name +
-            '">' +
-            '<div class="box box-success">' +
-            '<div class="box-header with-border no-user-select">' +
-            '<h3 class="box-title">' +
-            topic.description +
-            " (<code>" +
-            topic.name +
-            "</code>)" +
-            "</h3>" +
-            "</div>" +
-            '<div class="box-body">' +
-            '<div class="row" id="advanced-content-' +
-            topic.name +
-            '-body">' +
-            '<div class="col-xs-12 settings-container" id="advanced-content-' +
-            topic.name +
-            '-flex"></div>' +
-            "</div>" +
-            "</div>" +
-            "</div>" +
-            "</div>"
-        );
+
+        $("#advanced-settings-tabs").append(`
+          <div id="advanced-content-${topic.name}" role="tabpanel" class="tab-pane fade">
+            <h3 class="page-header">${topic.description} (<code>${topic.name}</code>)</h3>
```

Comment:
```suggestion
            <h3 class="page-header">${topic.description}</h3>
```

I don't think `topic.name` adds much of a value to the header. Each box contains the full config path as header in case I need to know it.

Reply:
I didn't change that... I think it helps a little to understand the sections.

Reply:
Your right... I never noticed it - not even that there are sections at all in All Settings.